main.go

```diff
@@ -123,6 +126,13 @@ func main() {
 
 	var err error
 
+	//nolint:reassign // We need to reassign ErrorHandlers to register our handler
+	utilruntime.ErrorHandlers = append(utilruntime.ErrorHandlers, func(err error) {
+		if errors.Is(err, x509.UnknownAuthorityError{}) || errors.Is(err, x509.CertificateInvalidError{}) {
+			logger.Errorf(err, "Certificate error: %s", resource.ToJSON(err))
```

Comment:
I’m concerned this will result in huge amounts of logs, I’ll change this so that certificates are only logged once.